common/src/main/java/com/microsoft/identity/common/internal/providers/oauth2/OpenIDProviderConfigurationClient.java

```diff
@@ -0,0 +1,15 @@
+package com.microsoft.identity.common.internal.providers.oauth2;
+
+/**
+ * Created by shoatman on 11/17/2017.
+ */
+
+public class OpenIDProviderConfigurationClient {
+
+    public OpenIDProviderConfiguration getOpenIDProviderConfiguration(){
```

Comment:
If this call will make network calls, we may want to think about providing a callback mechanism or `Future` object to retrieve the result: if this work happens on a background-thread, returning just a result is fine and and network requests shouldn't be a problem.

Reply:
AsyncTask likely... but let's think about it